docs/articles/documentation/test-api/selecting-page-elements/selector-options.md

```diff
@@ -61,10 +83,10 @@ returns one or several nodes. If none of these nodes match the `text` filter, th
 
 You can also use the [index](#optionsindex) option to identify a node by its index.
 
-For details, see [Initializers that Return Multiple Nodes](selectors.md#initializers-that-return-multiple-nodes).
-
 ## options.index
 
+> Important! ***Deprecated.*** *Use the [nth()](selectors.md#filter-multiple-dom-nodes) method to select a node by its index. The [options.index](#optionsindex) option will be removed in future releases.*
+
```

Comment:
> Use the index option when a function or CSS selector string that was used to initialize the selector returns more than one DOM node. Otherwise, this option will be ignored.

It's not ignored anymore, see note in https://github.com/DevExpress/testcafe/pull/960